src/classes/client.ts

```diff
@@ -91,6 +93,14 @@ export class AIHordeClient extends Client {
 		this.horde_style_categories = res
 	}
 
+	async loadHordeCuratedLORAs() {
+		const source = this.config.data_sources?.curated_loras_source ?? `https://raw.githubusercontent.com/Haidra-Org/AI-Horde-image-model-reference/main/lora.json`
```

Comment:
i would use || instead of ?? in case the config has an empty string

Reply:
Done. I also updated categories and styles json files to do the same thing.
While I was at it, I noticed that the URLs still default to db0, so I fixed those as well.